.golangci.yml

```diff
@@ -0,0 +1,11 @@
+run:
+  skip-dirs:
+    - pkg/uroot/test
+
+# This enables a subset of the default linters
+linters:
+  disable-all: true
+  enable:
```

Comment:
You can get rid of stickler if you also enable revive and use this config:

https://github.com/u-root/uio/blob/main/.golangci.yml

(remove the file-header part though at the bottom, that can't apply in u-root at the moment, u-root is a little more complicated.)